opendatahub-io/odh-dashboard
# fix: remove old sass dependencies

## Description

Fix #859

## How Has This Been Tested?



1. Install this image: quay.io/lferrnan/odh-dashboard-test:new-build-deps
2. Check that is currently working

## Merge criteria:
